docs/cli/benchmark.mdx

```diff
@@ -19,20 +19,21 @@ This command is compatible with all OpenAI and OpenAI-compatible endpoints.
 ## Usage
 
 ```bash
-cortex benchmark
+cortex benchmark [options] [model_id]
 ```
 :::info
-To benchmark a specific model, edit `model.yaml` in `~/cortex/benchmark`. A feature to select models for benchmarking will be added soon.
+To benchmark a specific model, edit `model.yaml` in `~/cortex/benchmark`.
 :::
 
 
 ## Options
 
 | Option                           | Description                                                                 | Required | Default value |
 |----------------------------------|-----------------------------------------------------------------------------|----------|---------------|
-| `-n, --num_rounds <num_rounds>`  | Number of rounds to run the benchmark.                                       | No       | 10      |
+| `model_id`                | The model identifier you want to benchmark.                                                           | No       |      `Prompt to select from the available models`         |
+| `-n, --num_rounds <num_rounds>`  | Number of rounds to run the benchmark.                                       | No       | `10`      |
```

Comment:
lets separate the syntax highlighting to be more readable, 

i.e. `-n`, `--num_rounds`
The <input> is not necessary